integration-test/src/features/getFeesMulti.feature

```diff
@@ -0,0 +1,263 @@
+Feature: GetFees - Get List of fees by CI, amount, method, touchpoint
+
+  Background: 
+    Given the configuration "dataCart.json"
+
+  Scenario: Execute a GetFeesMulti request
+    Given initial json
```

Comment:
could be better something like that 
`Given GetFeesMulti request with json` ...